lib/bolt/node/winrm.rb

```diff
@@ -69,6 +69,12 @@ def shell_init
     $StdinInput = $Null
   )
 
+  if (-not (Get-Command $Path -ErrorAction SilentlyContinue))
+  {
+    $Host.UI.WriteErrorLine("Could not find executable '$Path' in ${ENV:PATH} on target node")
+    return 1
```

Comment:
2 things about this @joshcooper 

* Looking for your opinion on the security of dumping `PATH` - useful to end users or potential abuse vector? See commit message for example
* In testing this locally, discovered that the `return 1` is not actually behaving as expected due to how winrm is handling our `$LASTEXITCODE` set.  So in a situation where we fail to find the binary, the node still returns success - ticketing that separately and fixing in another PR to go up shortly